DirectEmployers/MyJobs
# [PD-1275] redirect user to main profile page upon registration or activation

# Summary

Users used to see the screen below after registering their account. Now, they will go straight to the central myprofile edit page, `/profile`.

![screen shot 2016-04-04 at 2 48 32 pm](https://cloud.githubusercontent.com/assets/15107331/14259360/84f50520-fa74-11e5-86a3-9afee0b157a7.png)

# Test

* Open up Chrome incognito
* Go here: https://goo.gl/6Rsk41
* Then go here: https://secure.my.jobs/
* Create an account with a fake email address
* You should be directed to: https://secure.my.jobs/profile/view/
